scripts/remove-root-items

```diff
@@ -3,7 +3,7 @@
 # remove-root-items
 # Author: Michael Stealey <michael.j.stealey@gmail.com>
 
-find /home/docker/hydroshare -user root -ls | rev | cut -d ' ' -f 1 | rev > root-items.txt
+find /hydroshare -user root -ls | rev | cut -d ' ' -f 1 | rev > root-items.txt
```

Comment:
@hyi at the moment yes, I am finding some things that do still get generated and owned by root and I'm trying to eliminate this. As of now, some log files are owned by root, and everything under the hydroshare/static directory is owned by root. Working on alpha now to debug these prior to completely removing this code.